velox/docs/functions/presto/math.rst

```diff
@@ -53,7 +53,12 @@ Mathematical Functions
 
 .. function:: from_base(string, radix) -> bigint
 
-    Returns the value of ``string`` interpreted as a base-``radix`` number. ``radix`` must be between 2 and 36.
+   Returns the value of ``string`` interpreted as a base-``radix`` number. ``radix`` must be between 2 and 36.
+
+.. function::   inverse_cauchy_cdf(median, scale, p) → double#
```

Comment:
This function needs to be documented with the rest of the inverse probability functions in the last section "Probability functions : inverse cdf" part of this file.